src/_scss/layouts/default/header/nav/_dropdownMenu.scss

```diff
@@ -21,6 +21,19 @@
         font-weight: $font-semibold;
         font-size: rem(15);
         line-height: rem(19);
+        .covid-newbadge {
+            height: rem(24);
+            width: rem(47);
+            border-radius: rem(4);
+            background-color: #b53332;
```

Comment:
Change to our closest pre-defined color variable, `$color-secondary-dark` (in `_newBadge.scss`)